tests/flow/test_persistency.py

```diff
@@ -203,3 +204,29 @@ def test04_repeated_edges(self):
             actual_result = graph.query(q)
             self.env.assertEquals(actual_result.result_set, expected_result)
 
+        # Test decoding of graph meta keys in different sequences.
+        def test05_decoding_order(self):
```

Comment:
Please test the following scenario
Present 50% of the virtual keys, after which, send `FLUSHALL` then retransmit all of the virtual keys.
`FLUSHALL` should perform clean-up of the graphs, including their decoding-context.